### ISSUE TITLE
Support Actions from other add-on (example: "Tab group" related actions)

### ISSUE THREAD

**Issue by klint** [Robbendebiene/Gesturefy#238]
Is it possible or would it be possible to assign actions from other add-on to Gesturify gestures? 

My current need is a gesture to open a Panarama-like view of open tabs in a group, like the "Simple Tab Groups" addon does (and a few other similar addons).

I have contacted the SimpleTabGroups developer, who would be ready to support this kind of request.
See https://github.com/Drive4ik/simple-tab-groups/issues/63#issuecomment-361288909

Thank you

**Comment by Robbendebiene** [comment]
Hey, this is a good idea and shouldn't be that hard to implement. However this feature only make sense if the user is able to create multiple gestures of the same action (https://github.com/Robbendebiene/Gesturefy/issues/197). Therefore I won't add this until the referenced feature is implemented.

**Comment by Drive4ik** [comment]
Hi @Robbendebiene 
In latest version of Simple Tab Groups I added functional to change groups from other extensions
But I can't to add your addon to white list because your addon don't have a static "Extension ID"

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/applications

When you add id, please tell me about this and I will add it to white list.

**Comment by Robbendebiene** [comment]
@Drive4ik Thanks for the information!
I'll let you know when I added the feature and updated my addon with an id.

**Comment by Robbendebiene** [comment]
This feature is now added in Gesturefy. The related command is called "Send message to other addon" and once @Drive4ik allows Gesturefy to communicate with his add-on I'll update the [wiki](https://github.com/Robbendebiene/Gesturefy/wiki/Send-message-to-other-addon) with an example.

**Comment by ldgbc** [comment]
@Drive4ik It cool that this feature is implemented.

Anyone know any interesting command to send to other add-on? Any way to find those command? or do we need to get the Add-on developer to support commands before we can use this feature?

The Wiki currently documents how to find IDs only.

**Comment by Drive4ik** [comment]
@ldgbc Hi! Gesturefy support this actions
https://github.com/Drive4ik/simple-tab-groups/blob/e67a19f7bd46a7a68666817c74e0ac7b391512e0/addon/src/js/constants.js#L52-L62

and please read this my comment https://github.com/Drive4ik/simple-tab-groups/issues/63#issuecomment-413834941

**Comment by Robbendebiene** [comment]
@ldgbc Yes, other add-ons need to support this. Currently I know about two:

- simple tab groups
- Tree Style Tab

But I'm pretty sure there are much more.

**Comment by ldgbc** [comment]
Guess I start complaining on other repo for it to be supported or ask for documentation if it already has.
